### PR TITLE
BRO-296 initial checkin of code

### PR DESCRIPTION
**Pull request by ReneStoeckmann**

### CODE REVIEW

File: documentation/scripts/SyntaxHighlighter/shAutoloader.js
Comment (rpbwp) [inline documentation/scripts/SyntaxHighlighter/shAutoloader.js:1]: Not a fan of shipping packed code, but I guess that's just how it is for this dependency. We could probably grab a new, readable version from here https://github.com/syntaxhighlighter/syntaxhighlighter

File: src/Kruso.Reepay.Extensions/Enums/ChargeStateEnum.cs
Comment (rpbwp) [inline src/Kruso.Reepay.Extensions/Enums/ChargeStateEnum.cs:1]: Not a fan of the way we use the enums here, we convert them `toString`  and then compare this string to the values returned by our API

**Review by rpbwp — APPROVED** [review]
Nothing obvious jumped at me security wise although we could definitely clean the code up and run a formatter.

lgtm

File: documentation/scripts/SyntaxHighlighter/shAutoloader.js
Comment (ReneStoeckmann) [reply documentation/scripts/SyntaxHighlighter/shAutoloader.js:1]: This is somehow autogenerated, but they did not provide us with the documentation project. So I would not touch the docs right now.

**Review by ivan-burlakov — APPROVED** [review]
I think we just need to merge as soon as it's initial checkin

**Pull request merged**